phi/tools/fal_tools.py

```diff
@@ -86,3 +89,39 @@ def generate_media(self, agent: Agent, prompt: str) -> str:
         except Exception as e:
             logger.error(f"Failed to run model: {e}")
             return f"Error: {e}"
+
+    def image_to_image(self, agent: Agent, prompt: str, image_url: Optional[str] = None) -> str:
+        """
+        Use this function to generate an image from a given image using the Fal AI API.
```

Comment:
What do you mean "From a given image". Maybe give a more detailed explanation and a link to their docs please.